src/data/content/design-system/fondamenti/dimensioni.yaml

```diff
@@ -0,0 +1,181 @@
+  twitterImage: null
+  canonical: null
+  pathname: /design-system/fondamenti/bordi-e-raggi/
+
+components:
+  hero:
+    background: null
+    title: Dimensioni
+    column: true
+    subtitle: "Le misure utilizzate per definire lo spazio occupato dagli elementi all'interno di un'interfaccia"
+    tag:
+      label: Fondamenti #C
+      addonClasses: mt-3 text-uppercase
+
+  # KANGAROO: METADATA
+    kangaroo:
+      id: kangarooHero
+      noPadding: true
+      titleSr: Metadati e link per approfondire
+      navposition:
+        items:
+          - title: Ti trovi in
+            tooltip: "La sezione di Designers Italia cui appartiene questa pagina"
+            icon:
+              icon: "sprites.svg#it-folder"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Fondamenti #C
+            url: "../../" #M
+      otherInfo:
+        items:
+          - title: Rispettando
+            tooltip: "Norme e riferimenti che i contenuti di questa pagina ti aiutano a rispettare"
+            icon:
+              icon: "sprites.svg#it-pa"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Linee guida di design #C
+            url: "/norme-e-riferimenti/linee-guida-di-design" #M
+      personalInfo:
+         items: #C
+          - title: Sei
+            tooltip: "Se non ti riconosci in questi profili non preoccuparti: potrai scoprire informazioni utili per saperne di più. In alternativa, naviga la sezione superiore per approfondire il contesto"
+            icon:
+              icon: "sprites.svg#it-user"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Designer, Developer, Qui per curiosità #C
+          - title: Vuoi
+            tooltip: "Obiettivi di progettazione per cui sono utili i contenuti di questa pagina"
+            icon:
+              icon: "sprites.svg#it-flag"
+              size: "sm"
+              align: "middle"
+              color: "secondary"
+            label: Progettare e sviluppare l'interfaccia utente #C
+      tagsLabel: Parliamo di
+      color: secondary
+      icon:
+        icon: sprites.svg#it-horn
+        size: sm
+        color: secondary
+      tags: [Progettazione interfaccia, Sviluppo interfaccia]
+  
+  sectionsEditorial:
+    - full: true
+      noSpace: false
+      paddingLeft: true
+      title: null
+      text: null
+      background: null
+      components:
+        - name: TextImageCta
+          title: A cosa servono
+          headingLevel: 2
+          text: |
+            Le dimensioni di un elemento sono misure che **definiscono lo spazio occupato** in altezza e in larghezza da un elemento.
+
+            Definendo dimensioni standard, infatti, è possibile garantire una coerenza visiva all'interno di un'interfaccia.
+
+            Le misure sono espresse in **percentuale** perché possono essere relative a: 
```

Comment:
```suggestion
            Le misure delle dimensioni sono espresse in **percentuale** e possono essere relative a: 
```

Reply:
@zetareticoli 
1. qui non so se è più corretto parlare di "misure" o "proprietà" delle dimensioni
2. il fatto che siano in percentuale deriva dal fatto che sono relative a? (il senso del "perché" che ho tolto insomma, e che ho sostituito, se sei d'accordo, con "e")